packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeButtons.tsx

```diff
@@ -36,17 +38,19 @@ export interface ModelsTreeHeaderButtonProps extends TreeHeaderButtonProps {
  * Custom hook that creates props required to render `ModelsTreeComponent` header button.
  * @public
  */
-export function useModelsTreeButtonProps({
-  imodel,
-  viewport,
-}: {
-  imodel: IModelConnection;
-  viewport: Viewport;
-}): Pick<ModelsTreeHeaderButtonProps, "models" | "viewport"> {
+export function useModelsTreeButtonProps({ imodel, viewport }: { imodel: IModelConnection; viewport: Viewport }): {
```

Comment:
same as with `useCategoriesTreeButtonProps`, I think it would be great to have a small code example in function docs